test/epgsql_cast.erl

```diff
@@ -40,9 +40,9 @@ await_connect(Ref) ->
         {C, Ref, connected} ->
             {ok, C};
         {_C, Ref, Error = {error, _}} ->
-            Error;
-        {'EXIT', _C, _Reason} ->
```

Comment:
This clause was removed because it looks like the order of reply message and 'EXIT' message isn't guaranteed